src/IntlProvider/locales/ar_EG.ts

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+
+exports.__esModule = true;
+exports.default = void 0;
+
+var _extends2 = _interopRequireDefault(require("@babel/runtime/helpers/extends"));
+
+var Calendar = {
+  sunday: 'حد',
+  monday: 'ثن',
+  tuesday: 'ثل',
+  wednesday: 'اع',
+  thursday: 'خم',
+  friday: 'جم',
+  saturday: 'سب',
+  ok: 'حسناً',
+  today: 'اليوم',
+  yesterday: 'أمس',
+  hours: 'ساعات',
+  minutes: 'دقائق',
+  seconds: 'ثواني',
+  formattedMonthPattern: 'MMM, YYYY',
+  formattedDayPattern: 'MMM DD, YYYY'
+};
+var _default = {
+  Pagination: {
+    more: 'المزيد',
+    prev: 'السابق',
+    next: 'التالي',
+    first: 'الاول',
+    last: 'الاخير'
+  },
+  Table: {
+    emptyMessage: 'لا يوجد بيانات',
+    loading: 'جاري التحميل...'
+  },
+  TablePagination: {
+    lengthMenuInfo: '{0} / صفحة',
+    totalInfo: 'الاجمالي: {0}'
+  },
+  Calendar: Calendar,
+  DatePicker: (0, _extends2.default)({}, Calendar),
+  DateRangePicker: (0, _extends2.default)({}, Calendar, {
+    last7Days: 'اخر 7 ايام'
+  }),
+  Picker: {
+    noResultsText: 'لا يوجد نتائج',
+    placeholder: 'إختيار',
+    searchPlaceholder: 'بحث',
+    checkAll: 'الجميع'
+  },
+  InputPicker: {
+    newItem: 'عنصر جديد',
+    createOption: 'إنشاء خيار "{0}"'
+  },
+  Uploader: {
+    inited: 'البدء',
+    progress: 'جاري الرفع',
+    error: 'خطأ',
+    complete: 'تم الإنتهاء',
+    emptyFile: 'فارغ',
+    upload: 'رفع'
+  }
+};
+exports.default = _default;
+module.exports = exports.default;
```

Comment:
Thank you for adding Arabic.
You should change the file to ES Module.

Reply:
> Thank you for adding Arabic.
> 
> You should change the file to ES Module.